charts/v2.15/cray-hms-hmcollector/Chart.yaml

```diff
@@ -8,6 +8,6 @@ sources:
 maintainers:
   - name: Hardware Management
     url: https://github.com/orgs/Cray-HPE/teams/hardware-management
-appVersion: "2.22.0"
+appVersion: "2.22.1"
```

Comment:
There is no 2.22.1 tagged version of the hms-hmcollector container image, and I don't see any pending PRs in the https://github.com/Cray-HPE/hms-hmcollector repository. 
```
[~]$ docker pull artifactory.algol60.net/csm-docker/stable/hms-hmcollector:2.22.1
Error response from daemon: manifest for artifactory.algol60.net/csm-docker/stable/hms-hmcollector:2.22.1 not found: manifest unknown: The named manifest is not known to the registry.
```

Did you mean the bump the chart version instead of the application version?

Reply:
This is is the location to update the version: https://github.com/Cray-HPE/hms-hmcollector-charts/blob/main/charts/v2.15/cray-hms-hmcollector/Chart.yaml#L3

Reply:
Also please update the changelog: https://github.com/Cray-HPE/hms-hmcollector-charts/blob/main/changelog/v2.15.md